Psychtoolbox/PsychCal/MeasureDpi.m

```diff
@@ -168,3 +197,17 @@
     sca;
     psychrethrow(psychlasterror);
 end
+
+function [unitInches, units, unit] = EnglishOrSI()
+% Ask user whether to use the International System of Units or the English
```

Comment:
Please fix the code indentation for this function to be consistent within the function, and with the other code.